docs/contributing/technical/resolve-merge-conflicts.md

```diff
@@ -6,138 +6,79 @@ keywords:
 
-```shell
-git fetch origin --recurse-submodules=no --progress --prune
-git checkout main --
-git fetch upstream --recurse-submodules=no --progress --prune
-git merge upstream/main --no-stat -v
+```bash
+git diff <your-branch> upstream/main
 ```
 
-## Merge with `upstream`
-
-Then merge with the forked up-to-date `beta` (default branch):
-
-```shell
-git merge origin/main --no-ff -v
-```
-
-You will see something similar to:
-
-![proper merge but results in conflicts](../../../static/img/contributing-resolve-merge-conflicts-merge-conflicts.png)
+**NOTE**: Some of the OpenSauced repositories will use `main` for the main branch while others like the [app repository](https://github.com/open-sauced/app), will use `beta` for the main branch name.
 
-## Review Changes
+![git diff output review conflicts](../../../static/img/contributing-resolve-merge-conflicts-review-conflicts.png)
 
-To see what the changes are, run the command below:
+**NOTE**: If you have already opened up a pull request, then you can see if there is a conflict at the bottom of the PR. Even though it is possible to resolve conflicts through GitHub, it is best practice to resolve them locally.
 
-```shell
-git diff package.json
-```
-
-It will look like this:
+![merge conflicts git message](../../../static/img/contributing-resolve-merge-conflicts-dont-do.png)
 
-![review merge conflicts](../../../static/img/contributing-resolve-merge-conflicts-review-conflicts.png)
+## How to Resolve Merge Conflicts
 
-## Resolve Conflicts
+If you have merge conflicts, it is best to use a text editor to resolve them. Identify the conflicting files and open them up in your editor of choice. For the conflicting sections, you have an option to either keep the incoming changes, keep your changes or keep both sets of changes. The conflicting sections will be marked with `<<<<<<<`, `=======`, and `>>>>>>>` symbols.
 
-Since this pull request does not modify the `package.json` file, it is safe to fast-forward the changes from `origin/main`:
-
-```shell
-# overwrite with origin/main changes
-git show :3:package.json > package.json
-```
+Here is a [guide](https://dev.to/adiatiayu/how-to-resolve-merge-conflicts-using-the-merge-editor-feature-on-vs-code-pic) for an in depth walkthrough of the process. Once you have resolved all of the conflicts, then you can stage, commit and push your changes to your branch.
 
-A more traditional way of doing the same thing is:
-
-```shell
-# make a local copy of all changes and use --theirs
-# --theirs strategy overwrite with origin/main changes
-git show :1:package.json > base.package.json
-git show :2:package.json > branch.package.json
-git show :3:package.json > head.package.json
-git merge-file -p --theirs \
-    branch.package.json base.package.json head.package.json > package.json
+```bash
+git status
+git add .
+git commit -m "fix: resolving merge conflicts"
+git push
 ```
 
-## Commit Changes
+## How to Keep Your Branch Updated
```

Comment:
I think it makes more sense to move this section up as the first subsection because folks need to update their branch first before resolving conflicts.
What do you think?